src/Commands/SetShooter.h

```diff
@@ -11,8 +13,13 @@ class SetShooter : public frc::Command {
   bool IsFinished();
   void End();
   void Interrupted();
+
+  double GetSpeed() const;
```

Comment:
Functions should be organized alphabetically or  as in this case, by related functionality. That is the Getter and the Setter are next to each other.

Also, whitespace is great but we don't have to use every other line.